### ISSUE TITLE
Loading local models

### ISSUE THREAD

**Issue by yhtest1** [MaartenGr/KeyBERT#248]
Is it possible to load local embeddings model instead of downloading from Huggingface?

**Comment by MaartenGr** [comment]
Yes, you can load any local model using SentenceTransformers and then pass that model directly to KeyBERT.
